standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/metastore/conf/MetastoreConf.java

```diff
@@ -1363,7 +1366,7 @@ public enum ConfVars {
         "hive.metastore.stats.auto.analyze.worker.count", 1,
         "Number of parallel analyze commands to run for background stats update."),
     STORAGE_SCHEMA_READER_IMPL("metastore.storage.schema.reader.impl", "metastore.storage.schema.reader.impl",
-        DEFAULT_STORAGE_SCHEMA_READER_CLASS,
+        SERDE_STORAGE_SCHEMA_READER_CLASS,
```

Comment:
Does this work with other storage formats like ORC, parquet, and textfile? I see that SERDE_STORAGE_SCHEMA_READER is compatible with Iceberg which is awesome.

Reply:
Code flow 
https://github.com/apache/hive/blob/cdf34923122c5bd6f3fda8153a3e53b9ff051d88/standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/HMSHandler.java#L6300
will not go through storage serde for formats listed at 
https://github.com/apache/hive/blob/cdf34923122c5bd6f3fda8153a3e53b9ff051d88/standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/metastore/conf/MetastoreConf.java#L1304

I have validated for Avro and storage handler type formats like HBase, JDBC